src/components/LanguageWrapper/index.js

```diff
@@ -133,6 +118,7 @@ class LanguageWrapper extends Component {
         locale={lang}
         messages={messages}
         defaultLocale={DEFAULT_LANG}
+        textComponent={Fragment}
```

Comment:
it's defining the component to wrap React-Intl Components in (by default a span). This will allow the HOC <Format...> that React-Intl provides to export Fragments instead of wrapping things in spans. Check out the 'Call 311 or Submit an Online Request' title. no spans!